spec/integration/elastic_integration_spec.rb

```diff
@@ -1428,6 +1454,30 @@ def path; @path; end
           end)
       end
     end
+
+    describe 'when SSL enabled and verification is disabled' do
+
+      let(:settings) {
+        super().merge(
+          "ssl_enabled" => true,
+          "ssl_verification_mode" => "none"
```

Comment:
Before (without `NoopHostnameVerifier` when `ssl_verification_mode ` is none) were getting an exception:
```
logstash-1       |      Failure/Error: expect { subject.register }.not_to raise_error
logstash-1       |      
logstash-1       |        expected no Exception, got #<LogStash::ConfigurationError: Preflight check failed: Host name 'elasticsearch' does not match the certificate subject provided by the peer (CN=server, O=Logstash, L=Http Input, ST=NA, C=LS)> with backtrace:
logstash-1       |          # ./lib/logstash/filters/elastic_integration.rb:303:in `raise_config_error!'
logstash-1       |          # ./lib/logstash/filters/elastic_integration.rb:417:in `serverless?'
logstash-1       |          # ./lib/logstash/filters/elastic_integration.rb:347:in `initialize_elasticsearch_rest_client!'
logstash-1       |          # ./lib/logstash/filters/elastic_integration.rb:133:in `register'
logstash-1       |          # ./spec/integration/elastic_integration_spec.rb:1451:in `block in <main>'
logstash-1       |          # ./spec/integration/elastic_integration_spec.rb:1451:in `block in <main>'
logstash-1       |      # ./spec/integration/elastic_integration_spec.rb:1451:in `block in <main>'

```